flutter_frontend_server/lib/server.dart

```diff
@@ -111,13 +111,18 @@ Future<int> starter(
   frontend.ProgramTransformer transformer,
 }) async {
   ArgResults options;
-  frontend.argParser.addMultiOption(
-    'delete-tostring-package-uri',
-    help: 'Replaces implementations of `toString` with `super.toString()` for '
+  final bool handleDeleteToString =
+      !frontend.argParser.options.containsKey('delete-tostring-package-uri');
+  if (handleDeleteToString) {
```

Comment:
Won't this always evaluate to false?

Reply:
`frontend.argParser.options.containsKey('delete-tostring-package-uri')` will be true when this option is added to the front-end server in Dart SDK. At that point we should stop handling it here in the Flutter engine. This allows us to have a seamless migration.

Reply:
Ahhh I understand now, we're looking at a different arg parser we don't own. Ok, LGTM

Reply:
Perhaps as a nit add a TODO about when we can remove this with al ink to an issue